policyengine_us/parameters/gov/ed/pell_grant/efc/dependent/income_modification.yaml

```
@@ -0,0 +1,9 @@
+description: The Education Department counts this percent of a dependent student's income towards the expected family contribution.
+values:
+  2023-01-01: 0.5
+metadata:
+  unit: /1
+  label: Percent of dependent income for the dependent contribution
```

Include the program name in the label
```suggestion
  label: Pell Grant EFC dependent income assessment rate
```